chapel.html.markdown

```diff
@@ -929,17 +991,17 @@ proc main() {
     }
   }
 
-// A coforall loop will create a new task for EACH iteration.
+// A ``coforall`` loop will create a new task for EACH iteration.
 // Again we see that prints happen in any order.
-// NOTE: coforall should be used only for creating tasks!
+// NOTE: ``coforall`` should be used only for creating tasks!
 // Using it to iterating over a structure is very a bad idea!
   var num_tasks = 10; // Number of tasks we want
-  coforall taskID in 1..#num_tasks {
+  coforall taskID in 1..num_tasks {
```

Comment:
I know this is small, but I think using the count operator is meaningful here.

Reply:
Emmm...This seems of little importance to me since `#` has been introduced before.